gatsby-node.js

```diff
@@ -103,28 +122,49 @@ exports.onCreateNode = async ({node, actions, loadNodeContent, createNodeId, cre
 
 exports.createSchemaCustomization = ({actions: {createTypes}}) => {
     createTypes(`
-        type UserStoryMetadata {
-          build_tools: [String]
-          community_supports: [String]
-          company: String
-          company_website: String
-          industries: [String]
-          organization: String
-          platforms: [String]
-          plugins: [String]
-          programming_languages: [String]
-          project_funding: String
-          project_website: String
-          summary: String
-          team_members: [String]
-          version_control_systems: [String]
-        }
+    type UserStoryMetadata {
+      build_tools: [String]
+      community_supports: [String]
+      company: String
+      company_website: String
+      industries: [String]
+      organization: String
+      platforms: [String]
+      plugins: [String]
+      programming_languages: [String]
+      project_funding: String
+      project_website: String
+      summary: String
+      team_members: [String]
+      version_control_systems: [String]
+      authored_by: String
+      location: String
+      latitude: String
+      longitude: String
```

Comment:
```suggestion
```

Reply:
Why are these considered as a part of the `UserStoryMetadata` now? Any justification for it?